src/_data/sites/hi.yml

```
@@ -199,6 +200,7 @@ homepage:
     upcoming: आगामी संस्करण
     development: विकास संस्करण के तहत
     dateline: VERSION on DATE
+    version_support: Version Support
```

```suggestion
    version_support: संस्करण समर्थन
```

Are we not translating the Version support text into other languages? Is this intentional?

---

This is our standard practice when we add a new string because not everyone knows how to translate the strings. :smile:

I also think for things like this automatic translation is usually not that great, so I just leave it as English. We can certainly changes ones we are sure make sense.